context/app/static/js/components/workspaces/WorkspaceDetails/WorkspaceDetails.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { condenseJobs } from 'js/components/workspaces/utils';
+import JobLink from 'js/components/workspaces/JobLink';
+import JobStatus from 'js/components/workspaces/JobStatus';
+import { Flex } from './style';
+
+const typographyVariant = 'subtitle1';
+
+function WorkspaceDetails({ workspace }) {
+  const job = condenseJobs(workspace.jobs);
+
+  return (
+    <Flex>
+      <JobLink workspace={workspace} job={job} typographyVariant={typographyVariant}>
+        {workspace.name}
+      </JobLink>
+      <Typography variant={typographyVariant}>
+        <span>&nbsp;</span>|<span>&nbsp;</span>
```

Comment:
```suggestion
        &nbsp;|&nbsp;
```
Seems to work w/o spans?